### PR TITLE
Add Featured Speakers

### PR DESCRIPTION
**Pull request by alves-luis**
Based on the mockup, it seems to do the job. However, I'm not sure how we should handle the responsiveness of the component.

Should the images be evenly distributed across the width of the viewport or centered (as it is now)?

### CODE REVIEW

File: components/homepage/FeaturedImages.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+function FeaturedImages(props) {
+    return (
+            <Wrapper bgcolor="secondary.main">
+                <TitleBox xs="12">
+                    <Typography variant="h5">
+                        {props.title}
+                    </Typography>
+                </TitleBox>
+                <ImageBox container direction="row" justify="center" alignItems="center" spacing={4}>
```
Comment (herulume) [inline components/homepage/FeaturedImages.js]: Can't some of this be in a styled comp?

Comment (alves-luis) [reply components/homepage/FeaturedImages.js]: I believe not, as they are not CSS, but Grid props. You recommend using vanilla CSS (with the flexbox) over the material-ui Grid props?

**Comment by alves-luis** [comment]
Nothing against that, I was just following the mockup. Not sure if we want to overload the homepage with that much information. Maybe making the photo clickable to the page where we're listing all the speakers? And in that page, we would obviously include links to their social media.

If we were to include that in the homepage, I think that the hovering option would be the best, despite not being very mobile friendly

**Comment by miguelbrandao** [comment]
> I think that the hovering option would be the best, despite not being very mobile friendly

Oh yeah, I completely forgot about mobile. Scratch that. Maybe linking it to the actual speakers page would be the best option.

**Pull request merged**